Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cubeb-pulse"
+version = "0.0.1"
+authors = ["Dan Glastonbury <dglastonbury@mozilla.com>"]
+description = "Cubeb backed for Pulse Audio written in Rust"
```

Comment:
My plan was that each backend would live in it's own crate. That seems to be the rust way. (Many, small crates).